packages/reports/src/Containers/open-positions.tsx

```diff
@@ -192,17 +207,17 @@ const MobileRowRenderer = ({
     const progress_value = (getTimePercentage(server_time, date_start ?? 0, date_expiry ?? 0) /
         100) as TRangeFloatZeroToOne;
 
-    if (isMultiplierContract(type ?? '') || isAccumulatorContract(type ?? '')) {
+    if (isMultiplierContract(type ?? '') || isAccumulatorContract(type)) {
         return (
             <PositionsDrawerCard
                 contract_info={contract_info}
-                contract_update={contract_update}
-                currency={currency}
+                contract_update={contract_update as ContractUpdate}
```

Comment:
`contract_update={contract_update as ContractUpdate}` to `contract_update={contract_update}